spec/backtrace_spec.rb

```diff
@@ -146,5 +146,32 @@
         expect(described_class.parse(ex)).to eq(parsed_backtrace)
       end
     end
+
+    context "given an Oracle backtrace" do
+      let(:bt) do
+        [%(ORA-06512: at "STORE.LI_LICENSES_PACK", line 1945),
+         %(ORA-06512: at "ACTIVATION.LI_ACT_LICENSES_PACK", line 101),
+         %(ORA-06512: at line 2),
+         %(from stmt.c:243:in oci8lib_220.bundle)]
+      end
+
+      let(:ex) do
+        OCIError = AirbrakeTestError
+        OCIError.new.tap { |e| e.set_backtrace(bt) }
+      end
+
+      after { Object.__send__(:remove_const, :OCIError) }
+
+      let(:parsed_backtrace) do
+        [{ file: 'ORA-06512', line: 1945, function: 'STORE.LI_LICENSES_PACK' },
```

Comment:
I guess our best bet is to simply remove it, then.

Here's an error and type from such exception:
```
OCIError: ORA-02291: integrity constraint (STORE.SN_PACE_REGISTRATIONS_FK01) violated - parent key not found
```

OCIError is the type and the rest is the message.